system/core/pages.php

```diff
@@ -195,6 +210,24 @@ function loadPage(RequestContext $O_O) {
 					exit;
 				}
 				break;
+
+				case HyphaRequest::HYPHA_SYSTEM_PAGE_HELP: //bz help
+				if ($args[0] == 'help') {
+							echo getButtonInfo($args);
+						exit;
+				} else {
+					$hyphaHtml->find('#pagename')->text( __('help-pagetitle')); //		"help-pagetitle" => "Help index",
+					$hyphaPage = new helpPage($args);
+					//echo "pages: 222<br>\n";
+					if (count($args) == 2) if (hypha_isLanguage($args[1])) {//echo " -pages.php 222- " . $args[1];
```

Comment:
This should not stack two ifs in a single line, since now it is not really clear where the else belongs.